app/views/content/train-to-be-a-teacher/school-placements.md

```diff
@@ -0,0 +1,98 @@
+
+You'll need experience of teaching in 2 schools to be awarded [qualified teacher status](/train-to-be-a-teacher/what-is-qts).
+
+## Where will my placements take place?
+
+Where you complete your school placements will depend on whether you've chosen a primary or secondary teacher training course.
+
+Your training provider will select placement schools for you. They will contact you and discuss your situation to help them select a location that you can travel to.
+
+## What will I do on my placements?
+
+You school placements aims to give you enough experience to: 
+
+* meet all the [teachers’ standards](https://www.gov.uk/government/publications/teachers-standards)
+
+* achieve [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts)
+
+On your school placements, you'll get hands-on experience of what it's like to be a teacher. So, you’ll be spending a lot of time in the classroom learning how to teach lessons. Beyond the classroom, you can get involved in other aspects of school life, like school trips, school plays, parents' evenings and staff meetings.
+
+During your placements you’ll get to see your pupils progress and get an insight into how rewarding teaching can be.
+
+$abbe$
+
+## What support will I get during my placements?
+
+School placements can seem daunting, but there's lots of support available.
+
+### Your skills from training
+
+You’ll start your placements well equipped with a toolkit of theory and techniques that you'll learn in training. Placements are a chance to put your new skills into practice.
+
+### Mentors
+
+In each placement school, you'll have a dedicated mentor. Mentors are expert professionals and role models, who will give you knowledge, support and feedback throughout your placement to help you learn.
+
+Mentors also receive training on how to develop teachers and what trainees need to know to complete their course.
+
+$jennifer$
+
+### Tutors
+
+While your mentor is specific to each placement school, your tutor provides consistent support from your training provider throughout your placements. They'll work with you and your mentors to make sure you have enough support and are making good progress towards your placement goals.
+
+### School leadership and policies
+
+Your school leadership team and colleagues will be there to support you. Your placement schools may also have documents and policies in place to support their teachers. For example, all schools will have a behaviour policy that explains what's expected of pupils and what will happen if they misbehave.
+
+## Are placements paid?
+
+For fee-funded postgraduate training, you can [get financial support through student finance](/funding-and-support/tuition-fee-and-maintenance-loans), or [you may be eligible for bursaries or scholarships](/funding-and-support/scholarships-and-bursaries). This support is available for your whole training course, including school placements.
```

Comment:
Just flagging that we have two links pretty close together here - whilst this might technically be OK re accessibility, is it worth considering whether this is another way to do this @emmaachesongray ?